packages/@aws-cdk/region-info/build-tools/fact-tables.ts

```diff
@@ -196,6 +198,18 @@ export const APPMESH_ECR_ACCOUNTS: { [region: string]: string } = {
 
 // https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Lambda-Insights-extension-versions.html
 export const CLOUDWATCH_LAMBDA_INSIGHTS_ARNS: { [key: string]: any } = {
+  '1.0.178.0': {
+    x86_64: {
+      // Asia Pacific (Hyderabad)
+      'ap-south-2': 'arn:aws:lambda:ap-south-2:891564319516:layer:LambdaInsightsExtension:6',
+      // Asia Pacific (Jakarta)
+      'ap-southeast-3': 'arn:aws:lambda:ap-southeast-3:439286490199:layer:LambdaInsightsExtension:8',
+      // Europe (Spain)
+      'eu-south-2': 'arn:aws:lambda:eu-south-2:352183217350:layer:LambdaInsightsExtension:2',
+      // Middle East (UAE)
+      'me-central-1': 'arn:aws:lambda:me-central-1:732604637566:layer:LambdaInsightsExtension:6',
+    },
+  },
```

Comment:
Lets remove this and include only the new region registration in this PR. The lambda insights arns will be added in #23836, which has eu-central-2 when we register that too.

Reply:
Alright, didn't saw your PR, I remove my changes related to this 👍